### ISSUE TITLE
overview: "Store metrics" switch toggle needs a label

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#15190]
Currently, on the Overview page, the Store metrics switch does not have a label.

We could fix that by adding `aria-labelledby=""` attribute that points to the ID of the "Store metrics" label before it.

Additionally, it could benefit from being ported to PF4 (instead of our own custom switch that looks like PF4).
